### ISSUE TITLE
Does not work on Lenovo Yoga Book YB1-X90F

### ISSUE THREAD

**Issue by runderwo** [corbinlc/GNURootDebian#148]
Launching produces a console root shell in / with the rootfs and Android filesystems apparently properly merged, but nothing works and there's no network access.  Attempting to launch an xterm spews some errors on the console that are too quickly cleared to see, and then there is a message box about a failed VNC connection each time I try.  Running e.g. /usr/bin/perl manually works, but I cannot install anything nor figure out how to troubleshoot the network problem.

Note, this is an x86-64 Atom machine

**Comment by sergv** [comment]
Strange since I have exactly same tablet and major things like nemwork access work flawlessly.

**Comment by tomaklutfu** [comment]
If your network needs special dns ip address you must configure `/etc/resolv.conf` accordingly.

**Comment by runderwo** [comment]
You're right - I thought about this and realize that the default google public DNS that is configure won't work behind NAT.  This wasn't that obvious since utils like ping, host, etc are not available in the Debian image until installed by apt-get.  I could only echo the correct nameserver to /etc/resolv.conf to get going.

Any way in the default setup to help idiots like me that have no more than 5 minutes to spend getting something to run for someone else?

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam
